File: app/src/main/java/com/ljx/example/activity/MainActivity.java
package com.ljx.example.activity;

import android.os.Bundle;
import android.text.TextUtils;
import android.view.View;
import android.widget.TextView;
import android.widget.Toast;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.google.gson.Gson;
import com.ljx.example.R;
import com.ljx.example.model.User;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_login);
        User user = new User();
        user.setName("12");
        user.setSex("12");
        user.setAvatar("12");
        user.setInfo(new User.Info());
        user.setAge("12");
        System.out.println(user);
        if (TextUtils.equals(user.getSex(),"1")){
            System.out.println(user.getName());
        }
        String b = "{\"press\":\"126\",\"fence\":\"125\",\"reception\":{},\"privacy\":\"123\",\"pair\":\"124\"}";
        User c = new Gson().fromJson(b, User.class);
        System.out.println(c.getName());
        System.out.println(new Gson().toJson(c));
        String data = new Gson().toJson(user);
        System.out.println(data);
        TextView tv = findViewById(R.id.tv1);
        tv.setText(c.getName());
        tv.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Toast.makeText(v.getContext(),new Gson().toJson(c),Toast.LENGTH_SHORT).show();
            }
        });
    }
}

File: app/src/main/java/com/ljx/example/activity/MainActivity.kt
package com.ljx.example.activity

import android.os.Bundle
import android.util.Log
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
import com.ljx.example.IMyAidlInterface
import f.KtTopTest1
import com.ljx.example.R
import f.K
import f.i
import f.j
import f.k
import f.ktTopFun
import f.m
import f.toB

class MainActivity : AppCompatActivity() {

    val list = mutableListOf<IMyAidlInterface>()

    private val s = "test field"

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        Toast.makeText(this, K.h5, Toast.LENGTH_LONG).show()
        ktTopFun()
        val i = i
        val j = j
        val k = k
        val m = m
        m.toB
        KtTopTest1()
        test(this)
    }

    fun test(main: com.ljx.example.activity.MainActivity) {
        Log.d("LJX", main::class.java.name)
    }
}